subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc

```diff
@@ -34,7 +34,7 @@ If you are interested in migrating an existing Gradle build to the Kotlin DSL, p
 
 * The embedded Kotlin compiler is known to work on Linux, macOS, Windows, Cygwin, FreeBSD and Solaris on x86-64 architectures.
 * Knowledge of Kotlin syntax and basic language features is very helpful. The link:{kotlin-reference}[Kotlin reference documentation] and link:https://kotlinlang.org/docs/tutorials/koans.html[Kotlin Koans] will help you to learn the basics.
-* Use of the <<plugins#sec:plugins_block,plugins {}>> block to declare Gradle plugins significantly improves the editing experience and is highly recommended.
+* Use of the <<plugins.adoc#sec:plugins_block,plugins {}>> block to declare Gradle plugins significantly improves the editing experience and is highly recommended.
```

Comment:
Yes this was for standardization and also makes it easier to find links but I will do this type of cleanup is a separate PR since this is already monstrous.